org.theia_ide.theia_ide.metainfo.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.theia_ide.theia_ide</id>
+
+  <name>Theia IDE</name>
+  <summary>A modern and open IDE</summary>
+
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+
+  <developer id="org.eclipse">
+    <name>Eclipse Foundation</name>
+  </developer>
+
+  <branding>
+    <color type="primary" scheme_preference="light">#8ff0a4</color>
+    <color type="primary" scheme_preference="dark">#865e3c</color>
+  </branding>
+
+  <description>
+    <p>
+      The Theia IDE is a modern IDE built on the Theia Platform.
+    </p>
+    <p>
+      The Theia IDE is built upon the highly modular Theia platform, enabling the integration of custom extensions and the creation of fully tailored tools. It supports the VSCode API, thus it is compatible with most of the VSCode addons.
+    </p>
+  </description>
+
+  <!-- Generated with https://hughsie.github.io/oars/generate.html -->
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-info">mild</content_attribute>
+  </content_rating>
+
+  <releases>
+    <release version="1.60.200" date="2025-04-14">
+      <url type="details">https://eclipsesource.com/blogs/2025/04/10/eclipse-theia-1-60-release-news-and-noteworthy</url>
+    </release>
+  </releases>
+
+  <requires>
+    <control>keyboard</control>
+    <control>pointing</control>
+    <display_length compare="ge">768</display_length>
+  </requires>
+
+  <url type="bugtracker">https://github.com/eclipse-theia/theia-ide/issues</url>
+  <url type="homepage">https://theia-ide.org</url>
+  <url type="vcs-browser">https://github.com/eclipse-theia/theia-ide</url>
+
+  <launchable type="desktop-id">org.theia_ide.theia_ide.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/flathub/org.theia_ide.theia_ide/master/screenshot1.png</image>
```

Comment:
screenshot URL should never reference the tip of a branch.

Also this repo doessn't exist either.

Ideally screenshots would be upstream.

Reply:
The only official screenshot for Linux I could find is [this](https://theia-ide.org/static/blueprint-min-d01cc012c94ae596708e3fcb603493c3.png). It's pretty dated, both because of the system theme and because it's branded "Theia Blueprint", that is the old name.

What should I do to submit my screenshot?

Reply:
https://snapcraft.io/theia-ide has a screenshot, although it is a bit self-referential. To submit your own screenshot, upload it to a CDN and link it here.